testing/testcafe/tests/dataSources/testData.ts

```diff
@@ -0,0 +1,12 @@
+export const getData = (rowCount: number, colCount: number): Record<string, string>[] => {
```

Comment:
Let's move this function to the file with test cases:
```testing/testcafe/tests/dataGrid/editing.ts```

Or if you want to reuse this helper function in the future, let's place it in the next place:
```testing/testcafe/tests/dataGrid/helpers/generateDataSourceData.ts```